stats/stats-proto/build.rs

```diff
@@ -22,7 +22,8 @@ fn compile(
             "#[serde(default)]"
         )
         .field_attribute(".blockscout.stats.v1.Point.is_approximate", "#[serde(skip_serializing_if = \"std::ops::Not::not\")]")
-        .field_attribute(".blockscout.stats.v1.Point.is_approximate", "#[serde(default)]");
+        .field_attribute(".blockscout.stats.v1.Point.is_approximate", "#[serde(default)]")
+        .field_attribute(".blockscout.stats.v1.GetLineChartRequest.resolution", "#[serde(default = \"default_resolution_i32\")]");
```

Comment:
can you just leave `#[serde(default)]` without writing default function? i32 always defaults to 0 and unspecified option is always the first option, so I think no need to create `default_resolution_i32`